source/2020-10-01-hacktoberfest.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Hacktoberfest 2020
+author: Jen Weber
+tags: Recent Posts, Announcement
+responsive: true
+---
+
+Have you ever spotted something in Ember code or documentation that needed fixing?
+Did you want to get involved in contributing to open source, but you have no idea where to start?
+
+Good news! It's Hacktoberfest 2020, and you are invited to participate in Ember-focused special events and mentorship opportunities through the month of October.
+You can make some commits, meet some new people, and maybe get some free stuff!
+
+## What is Hacktoberfest?
+
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is an annual, worldwide celebration organized by DigitalOcean and sponsored by many companies that use open source software.
+
+The premise is simple: 
+- Sign up anytime between October 1 and October 31
+- Make four valid Pull Requests to _any_ public repositories on GitHub
```

Comment:
For small numbers, I think it's easier to read the number.

```suggestion
- Make 4 valid Pull Requests to _any_ public repositories on GitHub
```

Reply:
For small numbers, it is grammatically correct for them to be spelled out.